blob/fileblob/fileblob.go

```diff
@@ -160,7 +160,7 @@ func (b *bucket) NewTypedWriter(ctx context.Context, key string, contentType str
 	if strings.HasSuffix(path, attrsExt) {
 		return nil, fmt.Errorf("open file blob %s: extension %q is reserved and cannot be used", key, attrsExt)
 	}
-	if err := os.MkdirAll(filepath.Dir(path), 0777); err != nil {
+	if err := os.MkdirAll(filepath.Dir(path), os.ModePerm); err != nil {
```

Comment:
You shouldn't change `0777` to `os.ModePerm`.
This is a false positive from `namedConst` checker.